app/src/androidTest/java/com/github/se/project/ui/endToEnd/TutorEndToEnd.kt

```diff
@@ -151,7 +153,9 @@ class TutorEndToEndTest {
     composeTestRule.onNodeWithText("Physics Tutoring").assertIsDisplayed()
     composeTestRule.onNodeWithText("physics").assertIsDisplayed()
     composeTestRule.onNodeWithText("physics").performClick()
-    Thread.sleep(5000)
+    // Todo: replace the Thread.sleep with composeTestRule.waitForIdle() but not sure if it will
+    // work
+    composeTestRule.waitForIdle()
```

Comment:
This test class is still commented but I think that with your updates it should now pass the CI and we may want to un-commented it.